server/public/stylesheets/style.css

```diff
@@ -0,0 +1,8 @@
+body {
```

Comment:
@RonLek Since we are not using the backend for SSR, we won't need public folder.